### PR TITLE
Feat/add time limit

### PR DESCRIPTION
**Pull request by trunghlt**

### CODE REVIEW

File: actableai/classification/cross_validation.py [hunk 1]
```diff
@@ -91,6 +91,7 @@ def run_cross_validation(
     residuals_hyperparameters: Optional[dict],
     num_gpus: int,
     eval_metric: str,
+    time_limit: Optional[int],
```
Comment (axeng) [inline actableai/classification/cross_validation.py:94]: I believe that for a parameter to be optional it must have a default value

File: actableai/regression/cross_validation.py [hunk 2]
```diff
@@ -28,6 +28,7 @@ def run_cross_validation(
     residuals_hyperparameters: Dict[str, Dict],
     num_gpus: int,
     eval_metric: str,
+    time_limit: Optional[int],
```
Comment (axeng) [inline actableai/regression/cross_validation.py:31]: Needs default value

File: actableai/tasks/classification.py [hunk 3]
```diff
@@ -31,6 +31,7 @@ def run(
         residuals_hyperparameters: Dict,
         num_gpus: int,
         eval_metric: str,
+        time_limit: Optional[int],
```
Comment (axeng) [inline actableai/tasks/classification.py:34]: Needs default value

File: actableai/tasks/regression.py [hunk 4]
```diff
@@ -31,6 +31,7 @@ def run(self,
             residuals_hyperparameters:Dict,
             num_gpus:int,
             eval_metric:Dict,
+            time_limit:Optional[int],
```
Comment (axeng) [inline actableai/tasks/regression.py:34]: Needs default value

File: actableai/tasks/regression.py [hunk 5]
```diff
@@ -221,6 +224,7 @@ def run(
             presets (str): _description_
             model_directory (str): _description_
             num_gpus (int): _description_
+            time_limit: time limit of training.
```
Comment (axeng) [inline actableai/tasks/regression.py:227]: Needs type

**Review by axeng — APPROVED** [review]

File: actableai/classification/cross_validation.py (hunk 1 above)
Comment (trunghlt) [reply actableai/classification/cross_validation.py:94]: it should be consistent wither other varibles as all are required here.

Comment (axeng) [reply actableai/classification/cross_validation.py:94]: Then they should not be optional I think

File: actableai/tasks/regression.py (hunk 5 above)
Comment (trunghlt) [reply actableai/tasks/regression.py]: done

**Comment by trunghlt** [comment]
Please review again

**Pull request merged**